README.md

```diff
@@ -39,12 +39,21 @@ Assuming that you're starting from **this directory**:
 1. To build and run the web app, Docker needs to know about some environment variables. There are examples ones in `dev.env` - copy it to `.env` and fill in the correct values for you. Save the file and then run `source .env`.
 1. Install [Docker](https://docker.com)
    1. Also, For windows, click [here](https://wslstorestorage.blob.core.windows.net/wslblob/wsl_update_x64.msi) to install the WSL 2 (Windows Subsystem for Linux, version 2)
+ main
```

Comment:
Thank you! I think this and the `main` on line 56 are left over from the merge conflict resolution. When you remove those, this will be good to go.